pkg/systemd/ssh-list-host-keys.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -eu
+port="22"
+host="localhost"
+
+parse_addr() {
+    if [ -n "$1" ]; then
+        count=$(echo "$1" | grep -o ":" | wc -l)
```

Comment:
Simplified it by using bashisms. Not sure the trade off is worth it, but see what you think.